pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -5815,16 +5820,501 @@ func Test_server_MultiInsert(t *testing.T) {
 				},
 			}
 		}(),
+		func() test {
+			req := genMultiInsertReq(Float, vector.Gaussian, 100, f32VecDim)
+
+			return test{
+				name: "Decision Table Testing case 4.1: Fail to MultiInsert with 2 existed ID when SkipStrictExistCheck is false",
+				args: args{
+					ctx:  ctx,
+					reqs: req,
+				},
+				fields: fields{
+					name:    name,
+					ip:      ip,
+					svcCfg:  defaultF32SvcCfg,
+					svcOpts: defaultSvcOpts,
+				},
+				beforeFunc: func(t *testing.T, s *server) {
+					vecs := genF32Vec(vector.Gaussian, 2, f32VecDim)
+					for i, j := 49, 0; i < 51; i, j = i+1, j+1 { // start from the middle of insert request
```

Comment:
As I mentioned before PR, it is better to avoid using a magic number.
And, it may be not readable...